src/models/subgraph-entities/v2/distribute-payouts-event.ts

```diff
@@ -54,8 +54,8 @@ export const parseDistributePayoutsEventJson = (
   distributedAmount: j.distributedAmount
     ? BigNumber.from(j.distributedAmount)
     : undefined,
-  beneficiaryDistributionAmount: j.distributedAmount
```

Comment:
dumb bug in parsing json, was assigning the wrong value to `beneficiaryDistributionAmount`